hack/test-integration.sh

```diff
@@ -44,6 +97,13 @@ function waitForDeployment {
     return 1
   fi
 }
+function verifyPodIsSuccessfull {
+	podName=$(oc -n "$NAMESPACE" get po -lapp="route-monitor-operator" -ojsonpath='{.items[0].metadata.name}')
+	echo "status of the pod"
+	oc -n "$NAMESPACE" get po "$podName" -ojsonpath='{.status}'
+	echo "pod logs"
+	oc -n "$NAMESPACE" logs "$podName"
```

Comment:
We could maybe pipe it to in the end.
| egrep "^|error|warning"
Not important at all, but why not

Reply:
not gonna fix as there might be more interesting info in the logs

Reply:
yes this egrep statement doesn't filter but only highlights the error and warning statements.
But doesn't matter. I am fine with how it is